### PR TITLE
Adressing issue: Rename apps/Llama2App.py to follow snake case conven…

### LINKED ISSUE #665: Rename apps/Llama2App.py to follow snake case convention
### 🐛 Describe the bug

listed file name should follow snake case convention.
Also handle the related imports of this file and make sure that everything works.

### PR DESCRIPTION
**Pull request by Ayesha-Imr**
…tion #665

## Description

The issue:
Rename apps/Llama2App.py to follow snake case convention #665
has been addressed.

## Fixes (issue)

The relevant file, class and variable names have been renamed to snake case and references to these names in other files have also been modified.

- [ ] Refactor (does not change functionality, e.g. code style improvements, linting)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #665

### CODE REVIEW

**Review by deshraj — APPROVED** [review]
Nice, thanks for fixing this. ❤️

**Comment by deshraj** [comment]
@Ayesha-Imr can you please fix the test? I think you have to make the change here: https://github.com/embedchain/embedchain/blob/main/embedchain/__init__.py#L7

File: embedchain/apps/llama2_app.py [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 
 @register_deserializable
-class Llama2App(App):
+class llama2_app(App):
```
Comment (deshraj) [inline embedchain/apps/llama2_app.py:11]: This change is not needed. We want to follow the title casing for the class names.

**Comment by Dev-Khant** [comment]
@Ayesha-Imr Can you please check the above conversations and resolve the conflict so we can review and merge?

**Comment by taranjeet** [comment]
Thanks for your contribution @Ayesha-Imr .
Will have to close this PR as there are so many merge conflicts and the code is not relevant now.

**Pull request closed without merging**